snakemake/README.md

```diff
@@ -10,6 +10,24 @@ Let's cook some Spätzle!
     source /your/path/to/venv/bin/activate
     pip install -r requirements.txt
 
+### Windows
+
+* git bash
+```git 
+cd D:/git/github/rl_institut/
+git clone git@github.com:rl-institut/workshop.git
```

Comment:
this won't work if the user did not set ssh locally, cloning using http is less error prone for new users

Reply:
That's right, we may should change this in the linux part too